vhdbuilder/packer/convert-osdisk-snapshot-to-sig.sh

```diff
@@ -21,6 +21,6 @@ echo "ManagedImageSharedImageGalleryId: /subscriptions/${SUBSCRIPTION_ID}/resour
 
 disk_snapshot_id="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${AZURE_RESOURCE_GROUP_NAME}/providers/Microsoft.Compute/snapshots/${ARM64_OS_DISK_SNAPSHOT_NAME}"
 az snapshot update --resource-group ${AZURE_RESOURCE_GROUP_NAME} -n ${ARM64_OS_DISK_SNAPSHOT_NAME} --architecture Arm64
-az sig image-version create --location $AZURE_LOCATION --resource-group ${AZURE_RESOURCE_GROUP_NAME} --gallery-name PackerSigGalleryEastUS \
+az sig image-version create --location $AZURE_LOCATION --resource-group ${AZURE_RESOURCE_GROUP_NAME} --gallery-name ${SIG_GALLERY_NAME} \
```

Comment:
Please revert this change, convert-osdisk-snapshot-to-sig.sh doesn't have env variable ${SIG_GALLERY_NAME}.  Ubuntu 1804 arm64 build fail (https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=58231230&view=logs&j=b03669c0-70b9-5993-d66f-d20387ecabc8&s=13a1e9be-10b9-5809-47f7-c38e14a56b67)